internal/webhook/csv.go

```diff
@@ -0,0 +1,110 @@
+package webhook
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/go-logr/logr"
+	opv1a1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"k8s.io/utils/ptr"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+const (
+	WebhookPath = "/mutate-operators-coreos-com-v1alpha1-clusterserviceversion"
+)
+
+var (
+	ManagedPkgNames = []string{
+		// base csvs
+		"ocs-operator",
+		"rook-ceph-operator",
+		"ocs-client-operator",
+		"mcg-operator",
+		"noobaa-operator",
+		"odf-csi-addons-operator",
+		"csi-addons",
+		"cephcsi-operator",
+		"odf-prometheus-operator",
+		"recipe",
+
+		// extended csvs
+		"ibm-storage-odf-operator",
```

Comment:
why do we need it here? you can scale down any csv which isn't odf-operator (can be queried from OLM set env easily) in the namespace since we are managing the ns 

the edge case is only operators installed in all namespaces and maybe we need to look at how to filter those out.

(or)

we might want to get this info during build time about all the csvs that we going to manage, wdyt?